docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
 executions.
 
 
+.. _`Defining the WBEM Server`:
+
+Defining the WBEM Server
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The target WBEM server  can be defined on the command line in several ways with
+the in the following arguments :
+
+1. Define the WBEM server characteristics with a set of general options
```

Comment:
I don't think we should call that "WBEM server characteristics". None of them defines any characteristics of the server. All they do is to define the information for connecting to the server.

Reply:
Changed to connection information

Reply:
NOTE: I do not like the work information.